invenio_jsonschemas/ext.py

```diff
@@ -127,7 +129,14 @@ def get_schema(self, path):
         if path not in self.schemas:
             raise JSONSchemaNotFound(path)
         with open(os.path.join(self.schemas[path], path)) as file_:
-            return json.load(file_)
+            schema = json.load(file_)
+
+            for transform in self.app.config['JSONSCHEMAS_TRANSFORM']:
+                if transform in JSONSCHEMAS_TRANSFORMATIONS:
```

Comment:
As mentioned above, this is a bit limiting, would be good to have import strings (like https://github.com/inveniosoftware/invenio-jsonschemas/blob/master/invenio_jsonschemas/ext.py#L172-L175)